bot/exts/evergreen/wtf_python.py

```diff
@@ -0,0 +1,151 @@
+        """Gets the content of README.md from the WTF Python Repository."""
+        failed_tries = 0
+
+        for x in range(FETCH_TRIES):
+            async with self.bot.http_session.get(f"{WTF_PYTHON_RAW_URL}README.md") as resp:
+                log.trace("Fetching the latest WTF Python README.md")
+
+                if resp.status == 200:
+                    self.raw = await resp.text()
+                    await self.parse_readme(self.raw)
+                    log.debug("Successfully fetched the latest WTF Python README.md, breaking out of retry loop")
+                    break
+
+                else:
+                    failed_tries += 1
+                    log.debug(
+                        f"Failed to get latest WTF Python README.md on try {x}/{FETCH_TRIES}. Status code {resp.status}"
+                    )
+
+        if failed_tries == 3:
+            action = Action.UNLOAD
+        else:
+            action = Action.LOAD
+
+        verb = action.name.lower()
+        ext = self.__class__.__name__
+
+        try:
+            action.value(self.bot, ext)
+        except (commands.ExtensionAlreadyLoaded, commands.ExtensionNotLoaded):
+            log.debug(f"Extension `{ext}` is already  is already {verb}ed.")
+        else:
+            log.debug(f"Extension {verb}ed: `{ext}`.")
+
+    async def parse_readme(self, data: str) -> None:
+        """
+        Parses the README.md into a dict.
+
+        It parses the readme into the `self.headers` dict,
+        where the key is the heading and the value is the
+        link to the heading.
+        """
+        table_of_contents = re.findall(
+            r"\[👀 Examples\]\(#-examples\)\n([\w\W]*)<!-- tocstop -->", data
+        )[0].split("\n")
+        table_of_contents = list(map(str.strip, table_of_contents))
+
+        for header in table_of_contents:
+            match = re.findall(r"\[▶ (.*)\]\((.*)\)", header)
+            if match:
+                self.headers.update(
+                    {
+                        match[0][0]: f"{BASE_URL}{match[0][1]}"
+                    }
+                )
+
+    def fuzzy_match_header(self, query: str) -> Optional[str]:
+        """Returns the fuzzy match of a query if its ratio is above 90 else returns None."""
+        match, certainty = process.extractOne(query, self.headers.keys())
+        return match if certainty > MINIMUM_CERTAINTY else None
+
+    @staticmethod
+    def make_embed(query: str, link: str) -> Embed:
+        """Generates a embed for a search."""
+        embed = Embed(
+            title=f"WTF Python Search Result For {query}",
+            colour=constants.Colours.dark_green,
+            description=f"[Go to Repository Section]({link})"
+        )
+        embed.set_thumbnail(url=f"{WTF_PYTHON_RAW_URL}images/logo.png")
+        return embed
+
+    @commands.command(aliases=("wtf",))
+    async def wtf_python(self, ctx: commands.Context, *query: str) -> None:
```

Comment:
Was there a reason you chose to use `*query` here instead of using a keyword-only argument (which discord.py uses for a "consume-rest" behaviour)?
```py
async def wtf_python(self, ctx: commands.Context, *, query: str) -> None
```
This will save you the line where you `join` them together. 
https://discordpy.readthedocs.io/en/latest/faq.html#why-do-my-arguments-require-quotes